covalent_slurm_plugin/slurm.py

```diff
@@ -136,6 +149,48 @@ async def _client_connect(self) -> Tuple[bool, asyncssh.SSHClientConnection]:
             The connection object
         """
 
+        if self.sshproxy:
+            try:
+                import oathtool
+            except ImportError:
+                raise RuntimeError(
+                    "To use 'sshproxy' options, reinstall the Slurm plugin as 'pip install covalent-slurm-plugin[sshproxy]'"
```

Comment:
Just nitpicking, I assume covalent needs to be restarted as well ?

Reply:
I don't think so, but I can double check

Reply:
confirmed server restart is not needed since it's imported at runtime